crypto/kyber/kyber512_ref.c

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
okay, this does avoid dynamic and/or over-allocating stack memory etc in each function. But still "funny" macrofied.